README.md

```diff
@@ -73,6 +73,53 @@ with alerter.simulation_alert("super dooper sim"):
 which will send "BigSims: super dooper sim has completed without error." via slack once
 `run_simulation()` has completed.
 
+## CLI Usage
+
+    Usage: simulert [OPTIONS] COMMAND [ARGS]...
```

Comment:
Thoughts about making this description more general so users are aware of the CLI then direct them to `simulert --help` for the actual options? Definitely keep the example though.

Reply:
I think that would make more sense